src/main/java/ladder/domain/Height.java

```diff
@@ -0,0 +1,33 @@
+package ladder.domain;
+
+public class Height {
+
+    private static final int MIN_HEIGHT_RANGE = 1;
+    private static final int MAX_HEIGHT_RANGE = 100;
+
+    private final int height;
+
+    public Height(String input) {
+        int number = changeInputToInteger(input);
+        validateNumberIsIntRange(number);
+        this.height = number;
+    }
+
+    private int changeInputToInteger(String input) {
+        try {
+            return Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException("숫자로 입력을 변환할 수 없습니다.");
+        }
+    }
+
+    private void validateNumberIsIntRange(int number) {
+        if (number < MIN_HEIGHT_RANGE || number > MAX_HEIGHT_RANGE) {
+            throw new IllegalArgumentException("입력된 높이가 정해진 규칙에 맞지 않습니다.");
```

Comment:
어떤 입력값으로 인해, 그리고 어떤 기준으로 실패했는지 오류메시지를 풍부화 해주면 좋습니다. 
```suggestion
            throw new IllegalArgumentException("입력된 높이는 "+MIN_HEIGHT_RANGE + " 이상, " + MAX_HEIGHT_RANGE + " 이하여야 합니다. 입력값 : " + number);
```

Reply:
와.. 오류 메시지를 구체화하고 싶어서 
커스텀 예외를 활용해볼까 고민을 정말 많이했는데 이렇게 간단하게 적용시킬 수 있네요!! 

왜 생각을 못했던 걸 까요.. 고민에 사로잡혀 시야가 정말 좁아져있던 것 같아요
감사합니다!!